### DISCUSSION TITLE
Option to report a website directly from extension?

### DISCUSSION THREAD

**Discussion by Fooftilly** [OhMyGuus/I-Still-Dont-Care-About-Cookies#6398] [Ideas]
Is it viable to have an option to report a website that are not supported directly from extension? For example, having Github issue filled, or at least written out by extension and then copy/pasted to new issue.

**Comment by inson1** [comment]
@Fooftilly there is button for it
![xrnr2cCY13](https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/assets/75314629/e221d24a-ba2d-494c-9b1e-3b6b6d1c5463)

**Reply by inson1** [reply]
@Fooftilly May you close it?
